docs/docs/conferences.md

```diff
@@ -0,0 +1,19 @@
+﻿---
```

Comment:
@zpao Good catch. I will remove it.
Not too long ago I tried to build the site on my Windows machine, but it failed repeatedly. I thought the change was easy enough though, obviously not. Thx!